be/src/exec/vectorized/hdfs_scanner.h

```diff
@@ -50,8 +51,7 @@ struct HdfsScanStats {
 
     int64_t get_cpu_time_ns() const {
         // TODO: make it more accurate
-        return expr_filter_ns + column_convert_ns + level_decode_ns + value_decode_ns + group_dict_filter_ns +
-               group_dict_decode_ns;
+        return expr_filter_ns + column_convert_ns + column_read_ns + reader_init_ns;
```

Comment:
The new cpu time are not overlapped?

Reply:
No, not overlapped. Hoever `column_convert/decode` are included in `column_read_ns`, so `column_read_ns` is more accurated.